BookReader/BookReader.js

```diff
@@ -627,7 +627,7 @@ BookReader.prototype.drawLeafsOnePage = function() {
         var topInView    = (leafTop >= scrollTop) && (leafTop <= scrollBottom);
         var bottomInView = (leafBottom >= scrollTop) && (leafBottom <= scrollBottom);
         var middleInView = (leafTop <=scrollTop) && (leafBottom>=scrollBottom);
-        if (topInView | bottomInView | middleInView) {
```

Comment:
I need to review this. There's a chance the original developers could have intended to use `|`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#Bitwise_OR

Maybe it has the same effect though. Will get back to you tomorrow.